openmetadata-service/src/main/java/io/argoproj/workflow/JSON.java

```diff
@@ -0,0 +1,460 @@
+/*
+ * Argo Workflows API
+ * Argo Workflows is an open source container-native workflow engine for orchestrating parallel jobs on Kubernetes. For more information, please see https://argoproj.github.io/argo-workflows/
+ *
+ * The version of the OpenAPI document: VERSION
+ *
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+package io.argoproj.workflow;
```

Comment:
The java SDK for Argo Workflows is generated automatically from some openapi files. Currently there is a parsing issue of the type `Instant` that we need to patch by shadowing their `JSON` implementation and passing a new custom adapter